src/hockeypuck/openpgp/io.go

```diff
@@ -53,8 +84,12 @@ func WritePackets(w io.Writer, key *PrimaryKey) error {
 	return nil
 }
 
-func WriteArmoredPackets(w io.Writer, roots []*PrimaryKey) error {
-	armw, err := armor.Encode(w, openpgp.PublicKeyType, nil)
+func WriteArmoredPackets(w io.Writer, roots []*PrimaryKey, options ...KeyWriterOption) error {
```

Comment:
A unit test that asserts the headers are added to the output would be nice to have.

Reply:
Thanks for the reminder :wink: I've added a quick unit test for it.